doc/usage/Flexible_Loads.md

```diff
@@ -0,0 +1,242 @@
+
+```pycon
+>>> auth = ("username", "password")
+>>> en.solve_load_flow(auth=auth)
+2
+>>> abs(load_bus3.res_voltages)
+array([243.66463933, 232.20612714, 233.55093129]) <Unit('volt')>
+```
+
+The flexible loads are loads that implement some basic controls such as $P(U)$, $Q(U)$ or $PQ(U)$.
+
+## $P(U)$ control
+
+Let's remove `load3` from the network and add a flexible load as a replacement. A flexible load
+is a normal `PowerLoad` with a `flexible_params` argument that takes a list of`FlexibleParameter`.
+
+We first create a `FlexibleParameter` using its class method `p_max_u_production`. It returns a
+flexible parameter instance that reduces the active production when the voltage is higher than
+`u_up` volts and stops the production when the voltage reaches `u_max`. The `s_max` argument
+defines the maximum allowed apparent power of the production plant. In the example below,
+`u_up=240 V`, `u_max=250 V` and `s_max=4 kVA`.
+
+After that, a flexible load representing a PV plant is created. Its apparent power is fixed at
+`[si, 0, 0]` VA with `si` a negative value (negative because it is production). Theses apparent
+powers define the maximum power this load can produce. The `flexible_params` argument takes a
+list of `FlexibleParameter` instances, one per phase. For the first phase, the $P(U)$ control is
+used. For the two other phases, there is no control at all thus the `constant` class method is
+used.
+
+As a consequence, the provided apparent power for phase `'a'` is the maximum that can be produced
+(potentially modified by the $P(U)$ control) and the provided apparent power for phases `'b'` and
+`'c'` is the desired production as the flexible parameter is defined as `constant`.
+
+```pycon
+>>> # Let's make the load 3 flexible with a p(u) control to reduce the voltages constraints
+... en.loads["load3"].disconnect()
+... fp = FlexibleParameter.p_max_u_production(u_up=240, u_max=250, s_max=4000)  # V and VA
+... flexible_load = PowerLoad(
+...     id="load3",
+...     bus=load_bus3,
+...     phases="abcn",
+...     powers=[si, 0, 0],  # W
+...     flexible_params=[fp, FlexibleParameter.constant(), FlexibleParameter.constant()],
+... )
+```
+
+The load flow can now be run again. You can see that the voltage magnitude has changed. Note that
+the voltage magnitude for phase `'a'` was 240 V above without the $P(U)$ control, thus the control
+has been activated in this run.
+
+```pycon
+>>> en.solve_load_flow(auth=auth)
+4
+>>> abs(load_bus3.res_voltages)
+array([243.08225748, 232.46046866, 233.62854073]) <Unit('volt')>
+```
+
+The actually produced power of the flexible load is a result of the computation and can be
+accessed using the `res_flexible_powers` property of the load. Here, one can note that:
+
+* The active power for the phase `'a'` is negative meaning production;
+* The actual value of this active power is lower that the one requested as the control was activated;
+* The power for phases `'b'` and `'c'` is 0 VA as expected.
```

Comment:
This section (starting from "Here, one can ...") should be moved below the code block that shows the example below